github/actions/multi_client.go

```diff
@@ -66,8 +66,10 @@ func (m *multiClient) GetClientFor(ctx context.Context, githubConfigURL string,
 	client, err := NewClient(
 		githubConfigURL,
 		&creds,
-		WithUserAgent(m.userAgent),
-		WithLogger(m.logger),
+		append([]ClientOption{
+			WithUserAgent(m.userAgent),
+			WithLogger(m.logger),
+		}, options...)...,
```

Comment:
do we need these many `...`? 😄

Reply:
You have to break them to variadic arguments by doing that
Only thing that can be done is to split it as:
```go
options := []ClientOption{
			WithUserAgent(m.userAgent),
			WithLogger(m.logger),
		}, opts...)
```
And then call it with `options...` 
I'm okay with both :smile:

Reply:
The idea was that that block wouldn't be changed very easily 😄 but yeah, what Nikola said :)